docs/source/user-manual/copy-paste-blocks.md

```diff
@@ -0,0 +1,40 @@
+---
+myst:
+  html_meta:
+    "description": "Copy, cut, and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:description": "Copy, cut, and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:title": "How to copy, cut, and paste Volto blocks"
+    "keywords": "Volto, Plone, frontend, React, User manual, edit blocks, copy, cut, paste"
+---
+
+(copy-cut-paste-blocks-label)=
+
+# Copy, Cut, and Paste blocks in Volto
+
+Volto supports copying and cutting of blocks from one page, and pasting them to another page.
+This chapter describes how to do so.
+
+(copy-cut-blocks-label)=
+
+## Copy/Cut blocks
+
+While in edit mode you can hold `ctrl` key to select multiple blocks you want to copy/cut.
+Once blocks are selected, you will have a copy <img alt="../_static/copy.svg" src="../_static/copy.svg" style="height: 1em;"> and a cut <img alt="../_static/cut.svg" src="../_static/cut.svg" style="height: 1em;"> option at the bottom of Toolbar.
+
+Additionally, Volto also has a hidden feature for selecting multiple blocks in sequence.
+This feature can be used by selecting a start block and an end block while holding the `shift` key.
+This will select all the blocks between the start and end blocks, allowing you to copy or delete multiple blocks at once.
+
+.. video:: ../_static/user-manual/blocks/block-copy-cut.mp4
```

Comment:
This needs to be in MyST syntax

````suggestion
```{eval-rst}
.. video:: ../_static/user-manual/blocks/block-copy-cut.mp4
```
````